nemo/collections/common/data/lhotse/dataloader.py

```diff
@@ -67,10 +69,12 @@ class LhotseDataLoadingConfig:
     quadratic_duration: float | None = None
     #   c. Lhotse bucketing.
     use_bucketing: bool = False
+    bucket_batch_size: list[int] | None = None
     num_buckets: int = 30
     num_cuts_for_bins_estimate: int = 10000
-    bucket_duration_bins: list[float] | None = None
+    bucket_duration_bins: Any = None  # list[float] | list[list[float]] | None = None
```

Comment:
Is there a reason why you must use Any instead of `list[float] | list[list[float]] | None`?

Reply:
OmegaConf doesn't allow unions of containers with explicit typing.